teuthology/salt.py

```diff
@@ -162,37 +169,40 @@ def __set_minion_master(self):
 
     def __start_master(self):
         """Starts salt-master.service on master_remote via SSH"""
-        self.master_remote.run(args = ['sudo', 'systemctl', 'restart',
-            'salt-master.service'])
+        self.master_remote.run(args=[
+            'sudo', 'systemctl', 'restart', 'salt-master.service'])
 
     def __stop_minions(self):
         """Stops salt-minion.service on all target VMs"""
-        self.cluster.run( args=['sudo', 'systemctl', 'stop',
-            'salt-minion.service'])
+        self.cluster.run(args=[
+            'sudo', 'systemctl', 'stop', 'salt-minion.service'])
 
     def __start_minions(self):
         """Starts salt-minion.service on all target VMs"""
-        self.cluster.run( args=['sudo', 'systemctl', 'restart',
-            'salt-minion.service'])
+        self.cluster.run(args=[
+            'sudo', 'systemctl', 'restart', 'salt-minion.service'])
 
     def __ping(self, ping_cmd, expected):
         with safe_while(sleep=5, tries=10,
-                action=ping_cmd) as proceed:
+                        action=ping_cmd) as proceed:
             while proceed():
                 output = StringIO()
-                self.master_remote.run(args = ping_cmd, stdout = output)
+                self.master_remote.run(args=ping_cmd, stdout=output)
                 responded = len(re.findall('True', output.getvalue()))
-                log.debug("{} minion(s) responded".format(responded))
+                log.debug('{} minion(s) responded'.format(responded))
                 output.close()
-                if(expected == responded):
+                if (expected == responded):
                     return
 
     def ping_minion(self, mid):
         """Pings a minion, raises exception if it doesn't respond"""
-        self.__ping(["sudo", "salt", mid, "test.ping"], 1)
+        self.__ping(['sudo', 'salt', mid, 'test.ping'], 1)
 
     def ping_minions(self):
-        """Pings minions with this cluser's job_id, raises exception if they don't respond"""
-        self.__ping(["sudo", "salt", "-C", "G@job_id:{}".format(self.job_id),
-            "test.ping"], len(self.remotes))
-
+        """
+        Pings minions with this cluser's job_id, raises exception if they
+        don't respond
+        """
+        self.__ping([
+            'sudo', 'salt', '-C', 'G@job_id:{}'.format(self.job_id),
+            'test.ping'], len(self.remotes))
```

Comment:
Wow, looks much nicer. Were all these cleanups necessary to get it to pass pep8?